grantnav/frontend/templates/terms.html

```diff
@@ -14,7 +14,8 @@ <h1>{% trans "Terms and conditions" %}</h1>
   
   <p> The term &#8216;you&#8217; refers to the user or viewer of this website.</p>
 
-  <p>The terms &#8216;360Giving&#8217; or &#8216;us&#8217; or &#8216;we&#8217; refers to the owner of the website. 360Giving's company registration number is 9668396. Its registered charity number is 1164883. The registered address is 360Giving, The Foundry, 17 Oval Way, London SE11 5RR.</p>
+  <p>The terms &#8216;360Giving&#8217; or &#8216;us&#8217; or &#8216;we&#8217; refers to the owner of the website. 360Giving's company registration number is 9668396. Its registered charity number is 1164883. The registered address is 360Giving, King's Place, 90 York Way, London, N1 9AG
+.</p>
```

Comment:
@morchickit was there a reason for putting this on a new line?

Reply:
nope. This is odd, I just changed the address, I didn't add a line